src/store/cockpitApi.ts

```diff
@@ -110,6 +145,7 @@ export const cockpitApi = emptyCockpitApi.injectEndpoints({
         GetBlueprintsApiResponse,
         GetBlueprintsApiArg
       >({
+        providesTags: ['Blueprints'],
```

Comment:
I think we need to add the following to `enhancedCockpitApi`. 

https://github.com/osbuild/image-builder-frontend/blob/3c27f68b975bb52053f0c82da0cae65cf289ffb1/src/store/enhancedImageBuilderApi.ts#L143C1-L195C7

I think we can move the `providesTags` there too